aliyun/src/main/java/org/apache/iceberg/aliyun/oss/OSSInputStream.java

```diff
@@ -51,6 +51,10 @@
   private final Counter<Long> readBytes;
   private final Counter<Integer> readOperations;
 
+  /**
+   * @deprecated moving to package-private in 0.15.0
+   */
+  @Deprecated
```

Comment:
Should the class itself also be moved to package-private?

Reply:
Yes, that would maintain parity with `GCSInputStream`, `EcsSeekableInputStream`, and `S3InputStream` as well.